assets/pages/new-contributions-landing/components/ContributionUsTicker.jsx

```diff
@@ -0,0 +1,111 @@
+// @flow
+
+// ----- Imports ----- //
+import React, { Component } from 'react';
+
+// ---- Types ----- //
+type StateTypes = {|
+  totalSoFar: number,
+  goal: number,
+|}
+
+type PropTypes = {}
+
+
+// ---- Helpers ----- //
+
+const getInitialTickerValues = () =>
+  fetch('https://interactive.guim.co.uk/docsdata-test/1ySn7Ol2NQLvvSw_eAnVrPuuRnaGOxUmaUs6svtu_irU.json')
+    .then(resp => resp.json())
+    .then((data) => {
+      const totalSoFar = parseInt(data.sheets.Sheet1[0].total, 10);
+      const goal = parseInt(data.sheets.Sheet1[0].goal, 10);
+
+      return { totalSoFar, goal };
+    });
+
+const percentageTotalAsNegative = (total: number, goal: number) => {
+  let percentage = ((total / goal) * 100) - 100;
+  if (percentage > 0) {
+    percentage = 0;
+  }
+  return percentage;
+};
+
+
+// ----- Component ----- //
+export class ContributionUsTicker extends Component<PropTypes, StateTypes> {
+
+  constructor(props: PropTypes) {
+    super(props);
+
+    this.state = {
+      totalSoFar: 0,
+      goal: 0,
+    };
+
+  }
+
+  componentDidMount(): void {
+    const self = this;
+    getInitialTickerValues().then(({ totalSoFar, goal }) => {
+      const initialTotal = totalSoFar * 0.8;
+      this.count = initialTotal;
+      this.totalSoFar = totalSoFar;
```

Comment:
I don't think there's any distinction between `this` and `self` in this case since arrow functions don't create a new `this` context. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#No_separate_this